modules/backend/behaviors/importexportcontroller/TranscodeFilter.php

```
@@ -17,7 +17,7 @@ class TranscodeFilter extends php_user_filter
 
     protected $encodingTo;
 
-    public function filter($in, $out, &$consumed, $closing)
+    public function filter($in, $out, &$consumed, $closing): int
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$closing'. (undefined)

(UnusedFormalParameter)